icechunk-python/python/icechunk/repository.py

```diff
@@ -67,6 +67,11 @@ def open(
 
         If no repository exists at the given storage location, an error will be raised.
 
+        !!! warning
+            In multiprocessing environments this method can end up in a deadlock, if multiprocessing start method of processes is set to 'fork', which is the default on POSIX systems except MacOS.
+            Change the start method to 'forkserver' is recommended: `mp.set_start_method('forkserver')`.
+            Read more in our [Parallel Write Guide](/icechunk-python/parallel#uncooperative-distributed-writes).
+
```

Comment:
I'm not sure this belongs in the docstring itself. I'd rather have this comment in the parallel guide.

Also see https://github.com/earth-mover/icechunk/pull/803

Reply:
As a user encountering such a deadlock without reading every page of the docs beforehand, I would start searching at the function which produces the deadlock, therefore I definitly would keep the warning with the link to the docs-page there. What do you think of this shortened version?

```
In multiprocessing environments this method can end up in a deadlock.
Read more in our [Parallel Write Guide](/icechunk-python/parallel#uncooperative-distributed-writes).
```

Reply:
Should I merge #803 into this branch?

Reply:
I'm sympathetic to adding it to the docstring. But if you had a deadlock problem, would you not look at the "parallel writes" page?

Reply:
Of course I would, but I wouldn't start there. Hence, a link there would be helpful and speed up my search for a solution.

Reply:
OK I've suggested a much briefer message, it's good to not duplicate very technical details in two places